llvm/lib/Target/AArch64/AArch64Arm64ECCallLowering.cpp

```diff
@@ -710,7 +710,7 @@ bool AArch64Arm64ECCallLowering::processFunction(
   // name (emitting the definition) can grab it from the metadata.
   //
   // FIXME: Handle functions with weak linkage?
-  if (F.hasExternalLinkage() || F.hasWeakLinkage() || F.hasLinkOnceLinkage()) {
+  if (!F.hasLocalLinkage() || F.hasAddressTaken()) {
```

Comment:
Instead of emitting the "arm64ec_unmangled_name" metadata here, and ignoring it later, can we just skip emitting it?

Reply:
I thought of that, but I felt it was worth keeping that metadata around in case it came in useful sometimes in the future.

Reply:
Addressed